### DISCUSSION TITLE
How to use GalleryFormMediaMixin so it won't affect original css styles?

### DISCUSSION THREAD

**Discussion by mateusz-grzegorzek** [dzhuang/django-galleryfield#46] [Q&A]
Hello all,
When I use `GalleryFormMediaMixin`
```
class GalleryFormMediaMixin:
    # todo: testcase (both settings and render)
    class Media:
        css = {
            "all": (conf.BOOTSTRAP_CSS_LOCATION,)
        }

        js = (
            conf.JQUERY_LOCATION,
            conf.BOOTSTRAP_JS_LOCATION
        )
```
like below
`class MyModelForm(GalleryFormMediaMixin, forms.ModelForm):`
all my original Admin Panel styling is overwritten.
Could you help me with fixing it?

**Comment by dzhuang** [comment]
It's a [known issue](https://django-galleryfield.readthedocs.io/en/latest/misc.html#known-issues) as listed in the docs. And I must say sorry I currently have no idea of how to fix that, though it only overrides the css styling of models related to galleryfield.

**Reply by mateusz-grzegorzek** [reply]
Thanks very much for the reply!

My guess is that it's importing below css file
`<link href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" type="text/css" media="all" rel="stylesheet">`
but no other solution than manually extract only necessary styling from it comes to my mind

**Reply by dzhuang** [reply]
> no other solution than manually extract only necessary styling from it comes to my mind

Exactly, because the bootstrap (conflicted) stuffs were loaded as form assets after those of the admin page.  Sorry for my late answer.
